assets/js/scrollDepth.js

```diff
@@ -1,19 +1,19 @@
 const SCROLL_DEPTH_LEVELS = [20, 40, 60, 80, 99];
 
 window.addEventListener('DOMContentLoaded', function () {
-    let observer
-    const el = document.getElementsByTagName('body')[0];
+  let observer
+  const el = document.getElementsByTagName('body')[0];
     
-    el.style.position = 'relative';
+  el.style.position = 'relative';
 
-    const initObserver = () => {
-      const cb = (entries) => {
-      const [IntersectionObserverEntry] = entries;
-      const { isIntersecting, target } = IntersectionObserverEntry;
-      const position = Number(target.dataset.depthMarkerId);
+  const initObserver = () => {
+    const cb = (entries) => {
+    const [IntersectionObserverEntry] = entries;
+    const { isIntersecting, target } = IntersectionObserverEntry;
+    const position = Number(target.dataset.depthMarkerId);
 
       if (isIntersecting) {
-       analytics.track('Scroll Depth', {
+        analytics.track('Scroll Depth', {
           scrollDepth: position === 99 ? 100 : position // triggering at 99% is more consistent than 100%.
```

Comment:
can position be floating pt? and should this instead be `position > 99 ? 100 : position`

Reply:
@davetroiano the position here is a hard coded data, not the scroll position it self.